dbt/adapters/base/impl.py

```diff
@@ -312,7 +311,13 @@ def _behavior_flags(self) -> List[BehaviorFlag]:
         """
         This method should be overwritten by adapter maintainers to provide platform-specific flags
         """
-        return []
+        return [
+            {
+                "name": "require_builtin_microbatch_strategy",
+                "default": False,
+                "description": "If True, then a builtin `microbatch` materialization strategy can be expected to exist.",
```

Comment:
The description is something that winds up in the user-facing warning; it's intended to be an explanation for the user of why they're seeing the warning. This sounds more like a dev-facing docstring. I think it could be more useful to provide a `docs_url` to the microbatch docs on docs.getdbt.com. You could then either update this or leave it with the default, which would be "The behavior controlled by `require_builtin_microbatch_strategy` is currently turned off."

Reply:
Goooootcha! Okay I'll update this to link to the documentation 🙂